src/storybook/utils/index.ts

```diff
@@ -0,0 +1,18 @@
+import { TokenDetails } from 'types'
+
+// searches for Token by symbol or address
+// accepts default in case none found
+export function findFromListHoC(tokens: TokenDetails[]): typeof _findToken {
+  function _findToken(symbolOrAddress: string): TokenDetails | undefined
+  function _findToken(symbolOrAddress: string, defaultToken: TokenDetails): TokenDetails
+  function _findToken(symbolOrAddress: string | undefined, defaultToken: TokenDetails): TokenDetails
```

Comment:
you call `findToken(undefined, defaultT)` somewhere?

Reply:
Yes, https://github.com/gnosis/dex-react/pull/1425/files#diff-d7d520ccdf63c7771001d0c9e57d5b23R39

since I provide the default, it was strange to me not to allow undefined too there. It was convenient